volume/stat.go

```diff
@@ -1,94 +1,51 @@
 package volume
 
 import (
-	"bufio"
 	"fmt"
-	"io"
 	"os"
 	"path"
-	"strconv"
 	"strings"
-	"time"
 
-	"github.com/puppetlabs/wash/munge"
 	"github.com/puppetlabs/wash/plugin"
 )
 
-// StatCmd returns the command required to stat all the files in a directory up to maxdepth.
-func StatCmd(path string, maxdepth int) []string {
-	// List uses "" to mean root. Translate for executing on the target.
-	if path == RootPath {
-		path = "/"
+func addAttributesForPath(dirmap DirMap, attr plugin.EntryAttributes, base, fullpath string, maxdepth int) {
```

Comment:
Thanks, this helped a bit. I think this could be `addEntryToDirMap`; `attr` => `entryAttr`, `fullpath` => `fullEntryPath` (or `absEntryPath` for consistency?), then `relative` => `relativeEntryPath`

Reply:
But none of the things it deals with are an Entry. Just the path representing one.

Reply:
Yeah, it's subjective. If I read that, I'd think of it as "addEntryToDirMap" => "where's the entry ...?" => "Oh the path is the entry. Ok this makes sense now".